index.html

```diff
@@ -9,6 +9,7 @@
 <body>
     <h1>Hello!</h1>
     <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Hic quam eligendi ipsum autem esse eaque laudantium cupiditate quibusdam! Sunt eius totam perspiciatis itaque sequi iure natus deleniti dicta, obcaecati cupiditate?</p>
+    <h2>Charles is the best!</h2>
```

Comment:
Hey Joe! This looks awesome, buuuut I think it should be an anchor tag and we can link to MY site!